### PR TITLE
Create herpetological-conservation-and-biology.csl

### PR DESCRIPTION
**Pull request by tersiphone**
This is a newly created CSL style for the journal Herpetological Conservation and Biology.  Please do not yet post to the repository as it will require final approve from our Board of Directors before being made publicly available.  I am posting here as a pull request so that it can be preliminarily reviewed for technical correctness.

The instructions to authors can be found here:

http://www.herpconbio.org/InstructionsForAuthors.pdf.

### CODE REVIEW

**Comment by tersiphone** [comment]
If I understand this failure correctly, it has to do with the fact that the ISSN and eISSN numbers indicated in the style are the same?  If that is the case, then I am a bit stuck because the journal only has an ISSN and not an eISSN.  I guess I can just remove the latter from the style?

If the failure was caused by something else, please let me know.

**Comment by adam3smith** [comment]
yes, the fields are not obligatory. If there's not a separate eISSN, just use the ISSN.

**Comment by tersiphone** [comment]
Done.

File: herpetological-conservation-and-biology.csl [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <category field="zoology"/>
+    <issn>1931-7603</issn>
+    <updated>2016-03-02T15:22:52</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author" suffix=".">
+      <name and="text" delimiter-precedes-last="always" et-al-min="10" et-al-use-first="10" initialize-with="." name-as-sort-order="first"/>
```
Comment (adam3smith) [inline herpetological-conservation-and-biology.csl]: set the `et-al` below in citation, so you only have to do this once. Also, you want the `et-al-min` to always be at least one bigger than the `et-al-use-first`. Having them at the same number is non-sensical (it'd imply using et al for an article with 10 authors and printing the first ten authors)

File: herpetological-conservation-and-biology.csl [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+      <label prefix=", "/>
+      <substitute>
+        <text macro="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name and="text" delimiter-precedes-last="always" et-al-min="10" et-al-use-first="10" initialize-with="." name-as-sort-order="first"/>
```
Comment (adam3smith) [inline herpetological-conservation-and-biology.csl]: see above

File: herpetological-conservation-and-biology.csl [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+      <label form="short" text-case="capitalize-first" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" delimiter="" and="text" et-al-min="3" et-al-use-first="1" initialize="false" initialize-with="" sort-separator=""/>
```
Comment (adam3smith) [inline herpetological-conservation-and-biology.csl]: move the `et al` here to `<citation>`

File: herpetological-conservation-and-biology.csl [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+        <text variable="number-of-pages" suffix=" p."/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="webpage">
+    <choose>
+      <if type="webpage">
+        <text variable="number" suffix=". "/>
+        <text variable="URL" prefix="Available from " suffix=". "/>
+        <text value="accessed" text-case="capitalize-first" prefix="[" suffix=" "/>
```
Comment (adam3smith) [inline herpetological-conservation-and-biology.csl]: why not `<text term...`?

Comment (adam3smith) [inline herpetological-conservation-and-biology.csl]: I'd wrap the accessed term and date in a group with `prefix="["` and `suffix="]"` It's easier to read than the two separated brackets, but it'll also not look weird in the (admittedly unlikely) case that the accessed date is missing.

**Comment by tersiphone** [comment]
Thanks @adam3smith.  I made all the changes you suggested, except the one "why not <text term...?"  I wasn't sure which term this was referring to.  Is it the "Accessed from"?  If so, I think I tried making that a term at first but it never rendered, which is why I made it a prefix.

Otherwise...?

File: herpetological-conservation-and-biology.csl [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+      </if>
+    </choose>
+  </macro>
+  <macro name="webpage">
+    <choose>
+      <if type="webpage">
+        <text variable="number" suffix=". "/>
+        <text variable="URL" prefix="Available from " suffix=". "/>
+        <group prefix="[" suffix="]">
+          <text value="accessed" text-case="capitalize-first" suffix=" "/>
```
Comment (adam3smith) [inline herpetological-conservation-and-biology.csl]: I meant this one: `<text term="accessed" text-case="capitalize-first" suffix=" "/>` should do exactly the same (and is more robust for future changes and localization to other languages)

**Comment by tersiphone** [comment]
Right.  Done.

**Comment by adam3smith** [comment]
(you can only do that with predefined terms, which is why "available from" didn't work. It's possible to do that, too, but it's a bit involved and I don't think it's worth the effort here. What you have now will work exactly as you want it to.)

**Comment by adam3smith** [comment]
Thanks! We really appreciate journals helping to provide their own styles, it's a huge help.

**Comment by tersiphone** [comment]
No worries.  Thanks for all your help!

What is the next step after I get editorial board approval for this to be hosted on the CSL and/or Zotero repository?

**Comment by adam3smith** [comment]
this will be on the Zotero repository within the hour and available through Mendeley within about 24hs. That happens automatically.

**Comment by tersiphone** [comment]
Yikes!!!  Can we prevent that from happening?  We need to get Board of Directors approval before it can be posted in the repositories.

**Comment by adam3smith** [comment]
I have taken the style down. Let me know when you want me to put it up again, I saved a copy.

**Comment by tersiphone** [comment]
Thanks.  Hopefully it will be in the coming days.

**Pull request merged**